### PR TITLE
update icds rate limiting

### PR DESCRIPTION
**Pull request by snopoke**
##### SUMMARY
add rate limit for multimedia downloads. apply rate limit for app downloads

##### ENVIRONMENTS AFFECTED
ICDS
##### ISSUE TYPE
- Change Pull Request
- Bugfix Pull Request

##### COMPONENT NAME
nginx

### CODE REVIEW

**Comment by snopoke** [comment]
already rolled out (though will different actual limits)

File: src/commcare_cloud/ansible/roles/nginx/vars/cas_ssl.yml [hunk 1]
```diff
@@ -82,6 +90,10 @@ nginx_sites:
        proxy_cache_lock_timeout: 30s
        client_body_buffer_size: 5m
        proxy_ignore_headers: Set-Cookie
+       limit_reqs:
+         - zone_name: "app_downloads"
```
Comment (emord) [inline src/commcare_cloud/ansible/roles/nginx/vars/cas_ssl.yml]: was this unused before?

File: src/commcare_cloud/ansible/roles/nginx/vars/cas_ssl.yml [hunk 2]
```diff
@@ -64,6 +68,10 @@ nginx_sites:
        proxy_next_upstream_tries: 1
        proxy_read_timeout: 900s
        proxy_cache: app_cache
+       limit_reqs:
+         - zone_name: "multimedia"
```
Comment (emord) [inline src/commcare_cloud/ansible/roles/nginx/vars/cas_ssl.yml]: From #server-fires it sounds like you also added something around limiting video files specifically? Or maybe that was just a suggestion of what we could do?

File: src/commcare_cloud/ansible/roles/nginx/vars/cas_ssl.yml (hunk 1 above)
Comment (snopoke) [reply src/commcare_cloud/ansible/roles/nginx/vars/cas_ssl.yml]: yes

File: src/commcare_cloud/ansible/roles/nginx/vars/cas_ssl.yml (hunk 2 above)
Comment (snopoke) [reply src/commcare_cloud/ansible/roles/nginx/vars/cas_ssl.yml]: Yes, we could split this out more. Another option would be to limit the bandwidth consumption here as well as just the requests per second:

https://nginx.org/en/docs/http/ngx_http_core_module.html#limit_rate

**Pull request merged**